ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/TemporaryEngine.java

```diff
@@ -125,8 +127,11 @@ public void givenKafkaTopic(final String name) {
         .preconditionTopicExists(name, 1, (short) 1, Collections.emptyMap());
   }
 
-  public PreparedStatement<?> prepare(final String sql) {
-    return getEngine().prepare(new DefaultKsqlParser().parse(sql).get(0));
+  public ConfiguredStatement<?> configure(final String sql) {
+    return ConfiguredStatement.of(
+        getEngine().prepare(new DefaultKsqlParser().parse(sql).get(0)),
+        new HashMap<>(),
```

Comment:
Would a Collections.emptyMap() better here?

Reply:
It needs to be mutable since some tests (e.g. SET PROPERTY) will modify this map.